lib/RTCSession.js

```diff
@@ -712,21 +637,19 @@ module.exports = class RTCSession extends EventEmitter
         debug('emit "sdp"');
         this.emit('sdp', e);
 
-        const offer = new RTCSessionDescription({ type: 'offer', sdp: e.sdp });
-
         this._connectionPromiseQueue = this._connectionPromiseQueue
-          .then(() => this._connection.setRemoteDescription(offer))
+          .then(() => mediaConnection.setRemoteSdp({type: 'offer', sdp: e.sdp}))
```

Comment:
We agreed that  `mediaConnection.setRemoteSdp()` would received an argument of type `RTCSessionDescription`.